src/rev_ai/custom_vocabularies_client.py

```diff
@@ -74,11 +83,7 @@ def delete_custom_vocabulary(self, id):
         :raises: HTTPError
         """
 
-        self._make_http_request(
-            "DELETE",
-            urljoin(self.base_url, id)
-        )
-
+        self._make_http_request("DELETE", urljoin(self.base_url, id))
```

Comment:
Are these line changes compatible with the python linter?

Reply:
Just curious why you made these reformatting decisions.

Reply:
The linter (I used `flake8`) did not flag anything wrong. Thought it was more simple and clean looking.